charts/ingress-nginx/Chart.yaml

```diff
@@ -16,6 +16,7 @@ engine: gotpl
 kubeVersion: ">=1.16.0-0"
 annotations:
   artifacthub.io/changes: |
+    - Add Default backend HPA autoscaling
     - Update jettech/kube-webhook-certgen image
```

Comment:
Please remove the other items on the list. Only your change should be there.

For context, this is used to render the changelog in the artifacthub page like this
![Screenshot from 2020-11-04 12-56-40](https://user-images.githubusercontent.com/161571/98360682-01af6b00-2009-11eb-8618-6e9bf223b6a5.png)

Reply:
Fixed.